src/tag/Tag.js

```diff
@@ -35,7 +85,9 @@ const Tag = (props) => {
       onClick={onClick}
       data-testid={dataTestId}
     >
+      {startIcon && <Icon icon={startIcon} size={size} />}
       <StyledTagLabel>{children}</StyledTagLabel>
+      {endIcon && <Icon icon={endIcon} size={size} />}
```

Comment:
Do we need startIcon or endIcon here? If we have either startIcon or endIcon the first block will render

Reply:
Really I dont need the second block at all. Everything should work with the first block and I should remove the conditional wrapping it. At one point when I was styling I had thought I needed two different returns to change the wrapper around the Tag. Didnt end up needing that but missed removing that extra code I put in.